### DISCUSSION TITLE
We should rename this repository to `analytical-platform`

### DISCUSSION THREAD

**Discussion by jasonBirchall** [ministryofjustice/analytics-platform#137] [Ideas]
There are too many `anayltics` and `analytical-platform` repositories. We should decide what the "correct" name is, rename this repository and then template new repositories using this name.

**Comment by roblevi** [comment]
May break a few things but I'm in favour

**Reply by jasonBirchall** [reply]
Do you have an idea about what's tied to the name of the repository? I'll do a GitHub search in our repositories to look for hard-coded examples.
